api/lib/opentelemetry/distributed_context/propagation/http_text_format.rb

```diff
@@ -3,6 +3,7 @@
 # Copyright 2019 OpenTelemetry Authors
 #
 # SPDX-License-Identifier: Apache-2.0
+require_relative './trace_parent'
```

Comment:
We've preferred to add `require`s at a higher level, rather than using `require_relative`. I don't know if there's any justification for that, though I recall reading somewhere that `require_relative` was "bad" in some way (somewhere on the internet, so it must be true). In this case, I'd add it to `api/lib/opentelemetry/distributed_context/propagation.rb`.

Reply:
sounds good